packages/angular/src/generators/mfe-host/schema.json

```diff
@@ -2,13 +2,13 @@
   "$schema": "http://json-schema.org/schema",
   "$id": "NxMFEHost",
   "cli": "nx",
-  "title": "Nx MFE Host App",
+  "title": "Nx MFE Host Application",
   "description": "Create an Angular Host Micro Frontend Application",
   "type": "object",
   "examples": [
     {
       "command": "nx g @nrwl/angular:mfe-host appName --remotes=remote1",
-      "description": "Create an Angular app with configuration in place for MFE. If remotes is provided, attach the remote app to this app's configuration."
+      "description": "Create an Angular application with configuration in place for MFE. If remotes is provided, attach the remote app to this app's configuration"
```

Comment:
```suggestion
      "description": "Create an Angular application with configuration in place for MFE. If the `remotes` option is provided, attach the remote application to this application's configuration."
```

Reply:
I left the period off the end of this because of how it renders on the docs site:
Note the `.:`

![image](https://user-images.githubusercontent.com/12140467/160836254-5ffec1ef-8278-43ea-b485-830449ccbf34.png)